content/2019-munich/schedule.md

```diff
@@ -122,11 +122,6 @@ title: Schedule
       <b>GitLab</b>
     </td>
   </tr>
-  <tr class="break">
-    <td>13:30</td>
-    <td>Lunch</td>
-    <td></td>
-  </tr>
   <tr class="talk">
     <td>14:30</td>
```

Comment:
This one needs a time update 14:30 -> 13:30 I believe.

Reply:
Fixed